lib/array.js

```diff
@@ -21,18 +21,20 @@ module.exports = (api) => {
     let count = 0;
     done = api.metasync.cb(done);
 
-    items.forEach((item, index) => {
-      fn(item, (err, value) => {
-        if (errored) return;
-        if (err) {
-          errored = true;
-          return done(err);
-        }
-        result[index] = value;
-        count++;
-        if (count === len) done(null, result);
-      });
-    });
+    const callback = (index, err, value) => {
```

Comment:
Rename `callback` to `next` because `callback` is a conventional name for incoming parameter